src/index.test.js

```diff
@@ -54,7 +54,7 @@ describe('Tmate GitHub integration', () => {
     Object.defineProperty(process, "platform", {
       value: "linux"
     })
-    core.getInput.mockReturnValue("false")
+    core.getInput.mockReturnValueOnce("false").mockReturnValueOnce("true").mockReturnValue("false")
```

Comment:
Why do we need _two_ additional mocked return values? From looking at the code, I would have expected only one additional one, just like in the hunk above.